scripts/linters/pylint_extensions.py

```diff
@@ -1772,45 +1772,37 @@ class BlankLineBelowFileOverviewChecker(checkers.BaseChecker):
         )
     }
 
-    def process_module(self, node):
-        """Process a module to ensure that there is a blank line below
+    def visit_module(self, node):
+        """Visit a module to ensure that there is a blank line below
         file overview docstring.
 
         Args:
             node: astroid.scoped_nodes.Function. Node to access module content.
         """
-
-        multi_line_indicator = b'"""'
-        file_content = read_from_node(node)
-        file_length = len(file_content)
-        triple_quote_counter = 0
-        empty_line_counter = 0
-        for line_num in python_utils.RANGE(file_length):
-            line = file_content[line_num].strip()
-            # Single line comment, ignore it.
-            if line.startswith(b'#'):
-                continue
-            triple_quote_counter += line.count(multi_line_indicator)
-
-            if line.endswith(b'"""') and triple_quote_counter == 2:
-                closing_line_index_of_fileoverview = line_num
+        # Check if the given node has docstring.
+        if node.doc is None:
+            return
+        line_number = node.fromlineno
```

Comment:
Why not use node.doc to find the number of lines in a docstring (docstring_lines) and then check `node.fromlineno + docstring_lines`? Will that be simpler? OR that's not possible?

also, using linenumber of the first child node in the model can be helpful?

Reply:
doc is not a node so we can't find the docstring line number directly, but there's an another way but I'm not sure about that -- we can get the line number of first import statement and then we can check if there is only one blank line above it. What do you think?

Reply:
hmmm.. that won't work if someone has added some code tO add python path for the third party imports, right? Can you please check whether we can make it simple using first child node of the module? (Else let's keep it as is!)